aws/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -66,8 +67,8 @@ data "template_file" "controller-configs" {
     etcd_domain = "${var.cluster_name}-etcd${count.index}.${var.dns_zone}"
 
     # etcd0=https://cluster-etcd0.example.com,etcd1=https://cluster-etcd1.example.com,...
-    etcd_initial_cluster = "${join(",", data.template_file.etcds.*.rendered)}"
-
+    etcd_initial_cluster   = "${join(",", data.template_file.etcds.*.rendered)}"
+    cgroup_driver          = "${local.channel == "edge" ? "systemd":"cgroupfs"}"
```

Comment:
Do we need to add support for flatcar edge in the same commit? Can this be done later, in a different PR? Or is it needed because of a bootkube dependency or minimize diff with typhoon?

Reply:
This is not needed since we have a better way of dealing with this in an automated way #62.